public/js/actions/breakpoints.js

```diff
@@ -33,8 +35,6 @@ function enableBreakpoint(location: Location) {
 function addBreakpoint(location: Location,
                        { condition, getTextForLine } : any = {}) {
   return ({ dispatch, getState, client }: ThunkArgs) => {
-    location = getOriginalLocation(getState(), location);
```

Comment:
I dropped these checks because it unecessary and frustrating to wrap in async. if we need them we can bring them back